src/App.css

```diff
@@ -107,16 +84,39 @@
   width: 96px;
 }
 
-.landing-page .overlap-wrapper {
+/* CSS Styling of Circles Right Side of Page */
+.box--two {
```

Comment:
If these are for the circles, perhaps the class name should be changed to reflect that better?